tasks.py

```diff
@@ -16,16 +16,17 @@ def requirements(ctx):
 
 @task
 def lint(ctx, fix=False):
-    ctx.run(
-        f"python -m black --target-version py36 {'--check' if not fix else ''} ./*.py ./src ./tests",
-        pty=True,
-    )
-
-    if fix:
-        print("\nSkipping type check as there is no fixer")
-    else:
-        print("\nRunning type check")
-        ctx.run("python -m mypy --strict src", pty=True)
+    skip_command = "echo 'Skipping'"
+    lint_commands = {
+        "isort": f"python -m isort {'' if fix else '--check-only --diff'} -y",
+        "black": f"python -m black {'' if fix else '--check'} .",
```

Comment:
interestingly black doesn't check for unused imports and the like. Do we need to add pylint for that?

Reply:
I wanted to, but relegated that to a different issue/pr